person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+
+  def can_use_services?
+    @age >= 18 || @parent_permission ? true : false
+  end
```

Comment:
- [ ] OPTIONAL: Any method that stays below private becomes a private method, instead of implicitly writing `private :of_age? ` you can just write private and put all private methods below, public methods on top: 
```
  def can_use_services?
    @age >= 18 || @parent_permission ? true : false
  end
  private
  def of_age?
    @age >= 18
  end

```